crates/eth-rpc/src/api/net_api.rs

```diff
@@ -0,0 +1,12 @@
+use jsonrpsee::core::RpcResult as Result;
+use jsonrpsee::proc_macros::rpc;
+use reth_primitives::U64;
+
+// TODO: Define and implement of methods of Net API
+#[rpc(server, namespace = "net")]
+#[async_trait]
+pub trait NetApi {
+    /// Returns the protocol version encoded as a string.
+    #[method(name = "version")]
+    fn protocol_version(&self) -> Result<U64>;
```

Comment:
for information, but why calling this with the `protocol_` prefix?

Reply:
This is how it [earlier](https://github.com/kkrt-labs/kakarot-rpc/blob/165d27a87fbd54a8430399c76af171c67511bb9f/crates/eth-rpc/src/servers/eth_rpc.rs#L41) was, but I have renamed it to version.